lighthouse-core/test/audits/metrics/total-blocking-time-test.js

```diff
@@ -7,31 +7,43 @@
 
 const TBTAudit = require('../../../audits/metrics/total-blocking-time.js');
 const defaultOptions = TBTAudit.defaultOptions;
+const constants = require('../../../config/constants.js');
 
 const trace = require('../../fixtures/traces/progressive-app-m60.json');
 const devtoolsLog = require('../../fixtures/traces/progressive-app-m60.devtools.log.json');
 
 const lcpTrace = require('../../fixtures/traces/lcp-m78.json');
 const lcpDevtoolsLog = require('../../fixtures/traces/lcp-m78.devtools.log.json');
 
-function generateArtifacts({trace, devtoolsLog, TestedAsMobileDevice}) {
+function generateArtifacts({trace, devtoolsLog}) {
   return {
     traces: {[TBTAudit.DEFAULT_PASS]: trace},
     devtoolsLogs: {[TBTAudit.DEFAULT_PASS]: devtoolsLog},
-    TestedAsMobileDevice,
   };
 }
 
-function generateContext({throttlingMethod}) {
-  const settings = {throttlingMethod};
-  return {options: defaultOptions, settings, computedCache: new Map()};
-}
+/**
+ * @param {{
+ * {LH.SharedFlagsSettings['formFactor']} formFactor
+ * {LH.SharedFlagsSettings['formFactor']} formFactor
+ * }} param0
+ */
+const getFakeContext = ({formFactor, throttlingMethod}) => ({
```

Comment:
sames

Reply:
(this one was already obj forma, but all good.)

Reply:
oh lol, sorry so it was :)